### PR TITLE
ref(native-stack-trace-v2): Add native stack trace new content design [NATIVE-343]

### PR DESCRIPTION
**Pull request by priscilawebdev**
- [x] Update the `render` logic of the ContentV3 to only be displayed when the platform is of type native
- [x] Update the ContentV3 file to render the new nativeFrame file + CSS updates
- [x] Create a new file called nativeFrame,  which renders the new design styled defined in the figma file https://www.figma.com/file/nZJDiuOgaLlU8pCSB1RjD4/Native-Stack-Trace?node-id=247%3A7318

**Before:**

_Preview_
image

_Issue Details_
image

**After:**

_Preview_

image

_Issue Details_

image

_Fix Empty Message Icon Size_

**Before:**
image

**After:**
image

ps: Every change is behind a feature flag

TODO: 
- [ ] Tests will be created in a follow-up PR after everything is approved and all feedback is applied
- [ ] Update how the `renderOmittedFrames` is rendered

### CODE REVIEW

**Comment by Jesse-Box** [comment]
Heyy!

So I looked at this and here are some things I would like tweaked.
- The grid-column-template(**large viewport only**): I think it will be best if we used fixed units for most row columns. `grid-template-columns: 24px 132px 116px 24px 1fr 24px;` worked well in most cases for me. I love the idea of a more flexible stack trace, but in practice the user will get less line length for the function & file column.
- Using `Roboto Mono` for the instruction address
- `gray300` for "Crashed, Called from" text as well as the file name
- The `AddressCell` has a `padding-left 8px;`which isn't needed (Not sure if this used for smaller viewports?)
- When the function and file name collapse onto 3 lines, the package name has some weird spacing issue that needs fixing. (See Below)
Screen Shot 2022-01-28 at 16 20 47

Excellent stuff @priscilawebdev !!

**Comment by priscilawebdev** [comment]
Thanks for the feedback @Jesse-Box 

. I updated the grid layout values but in the address column instead of `116px` I added `138px` as I noticed that some addresses were too long for the column and were breaking into multiple lines
. The Robot Font was there, but inside of a breakpoint. I've fixed it..thanks for pointing this out
. "Crashed, Called from"  have now the `gray300` color
. The `AddressCell` had 8px spacing on the left side because I thought it looked better with more spacing, but I've already removed it
. I updated the code to display less spacing. It looks like below:


image

**Comment by Jesse-Box** [comment]
Hey @priscilawebdev looks great!

Only one feedback point to give

Can you make the file paths next to the function (seen below) the same text colour as "Crashed in non-app" AND give a dotted line to indicate the tooltip behaviour.
Screen Shot 2022-02-01 at 10 29 52
e

File: static/app/components/events/interfaces/nativeFrame.tsx [hunk 1]
```diff
@@ -0,0 +1,460 @@
...
+  padding: ${space(1)};
+`;
+
+const ToggleButton = styled(Button)`
+  width: 16px;
+  height: 16px;
+`;
+
+const Package = styled('span')<{color?: Color}>`
+  border-bottom: 1px dashed ${p => p.theme.gray200};
```
Comment (matejminar) [inline static/app/components/events/interfaces/nativeFrame.tsx:447]:
```suggestion
  border-bottom: 1px dashed ${p => p.theme.border};
```

**Review by matejminar — APPROVED** [review]
The code looks good to me.

One observation:
Clicking on `Go to images loaded` takes you to the filtered down images loaded section. This section is collapsed by default nowadays. Should we consider opening it once the button is clicked?

**Comment by priscilawebdev** [comment]
@matejminar thanks!

Yes I will fix those regressions in a follow up PR

**Review by Jesse-Box — APPROVED** [review]

**Comment by priscilawebdev** [comment]
@Jesse-Box Thanks for the Feedback. I've updated the screenshots

**Pull request merged**